index.html

```diff
@@ -3054,30 +3054,38 @@ <h4>Semantic Interoperability</h4>
       <section>
         <h2>Integrity of Related Resources</h2>
         <p>
-When including a link to an external resource in a <a>verifiable credential</a>,
+When including a link to an external resource in a <a>conforming document</a>,
 it is desirable to know whether the resource that is pointed to is the same at
 signing time as it is at verification time. This applies to cases where there is
 an external resource that is remotely retrieved as well as to cases where the
-<a>issuer</a> and/or <a>verifier</a> may have local cached copies of a resource.
+<a>issuer</a> and/or <a>verifier</a> might have local cached copies of a
+resource.
         </p>
         <p>
 It is also desirable to know that the contents of the JSON-LD context(s) used in
-the <a>verifiable credential</a> are the same when used by both the
+a <a>conforming document</a> are the same when used by both the
 <a>issuer</a> and <a>verifier</a>.
         </p>
         <p>
-To validate that a resource referenced by a <a>verifiable credential</a> is the
+To validate that a resource referenced by a <a>conforming document</a> is the
 same at verification time as it is at issuing time, an implementer MAY include a
 property named <code id="defn-relatedResource">relatedResource</code> that
 stores an array of objects that describe additional integrity metadata about
-each resource referenced by the <a>verifiable credential</a>. If
+each resource referenced by the <a>conforming document</a>. If
 <code>relatedResource</code> is present, there MUST be an object in the array
-for each remote resource for each context used in the verifiable credential.
+for each remote resource used in the verifiable credential.
         </p>
         <p class="issue" title="Mandatory listing of contexts in relatedResouce are under debate.">
 The requirement that contexts be listed in `relatedResource` is currently being
 debated in the VCWG. This requirement might be removed in future iterations of
 the specification.
+        </p>
+        <p class="issue at-risk" title="Use of `relatedResource` in `VerifiablePresentation`">
```

Comment:
```suggestion
        <p class="issue at-risk" title="Use of `relatedResource` in `VerifiablePresentation` is at risk">
```

I think it is better to make the "at risk" features very explicit when reading the document. (I have not checked whether this is the case for the other, at risk features in the spec.)

Reply:
I got the "atrisk" class wrong and fixed it in de8355957fc69fe3b390caf93f48234bddf3fde5.

Which results in this rendering, does that work for you @iherman ?

![image](https://github.com/w3c/vc-data-model/assets/108611/f141b8a0-9789-40ec-94a2-2863295e85fd)